pkgs/test/test/runner/line_and_col_test.dart

```diff
@@ -0,0 +1,321 @@
+// Copyright (c) 20121, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Revert in this PR.

Reply:
Oh whoops not sure how this file got in here lol